pkg/logs/input/docker/launcher.go

```diff
@@ -75,6 +73,15 @@ func NewLauncher(sources *config.LogSources, services *service.Services, pipelin
 	return launcher, nil
 }
 
+func (l *Launcher) getCli() (*client.Client, error) {
```

Comment:
We probably want to call l.getCli() from https://github.com/DataDog/datadog-agent/pull/4839/files#diff-2325533c5d014118fdb0b16567c0b316L289

Reply:
Good catch. Thanks!
I've done the fix.